src/mfakto.cpp

```diff
@@ -136,6 +136,7 @@ int init_CLstreams(int gs_reinit_only)
         printf("ERROR: malloc(h_ktab[%d]) failed\n", i);
         return 1;
       }
+      memset(mystuff.h_ktab[i], 0, sizeof(mystuff.h_ktab[i]));
```

Comment:
This is wrong. If you want to zeroize the pointer, you can do
```
memset(&mystuff.h_ktab[i], 0, sizeof(mystuff.h_ktab[i]));
```
If you hand to zeroize the memory it points to, use
```
memset(mystuff.h_ktab[i], 0, sizeof(*mystuff.h_ktab[i]));
```

Reply:
Good catch! I want to zeroize the memory it's pointing to.